discordbot/stocks/options/options_menu.py

```diff
@@ -74,202 +73,95 @@ async def oi(
         await oi_command(ctx, ticker, expiration)
 
     @discord.ext.commands.command(name="stocks.opt")
-    async def opt(self, ctx: discord.ext.commands.Context, ticker=""):
-        """Stocks Context - Shows Options Menu"""
+    async def opt(self, ctx: discord.ext.commands.Context, ticker="", expiration=""):
+        """Stocks Context - Shows Options Menu
+
+        Returns
+        -------
+        Sends a message to the discord user with the commands from the stocks/options context.
+        The user can then select a reaction to trigger a command.
+        """
+
         if cfg.DEBUG:
             print(f"!stocks.opt {ticker}")
-        cols_temp = []
-        cols = []
 
-        if ticker == "":
+        if not ticker:
```

Comment:
I have hard times understanding some pythonic patterns
![image](https://user-images.githubusercontent.com/11668535/147269735-fe5f2407-6442-4569-9d0a-aec1a80aefc6.png)

Reply:
@piiq so when you do `a = ""`, `a` becomes a string. So when you compare `a` with `None` and `False` those are always false because you are comparing a string with a different variable type, in this case, the special type None and the boolean False. "_The “is keyword” is used to test whether two variables belong to the same object._"

However, when you do `not a` what happens is that the `not` makes it so you do the following `bool(a)` which in turn because it's empty by convention is `False`, and therefore returns a `True`.

Note:
```
bool(a) is False -> True
not a is True -> True
a is False -> False
a is None -> False
a is 0 -> False
```

Reply:
![image](https://user-images.githubusercontent.com/11668535/147271282-4bd44118-4e7c-4d29-9aa3-0aff289c7d4c.png)
right

Reply:
![Screenshot 2021-12-23 at 17 05 39](https://user-images.githubusercontent.com/25267873/147271750-ffaaec54-aa0a-46be-8540-ba2e61a9f63a.png)

Reply:
yep